modules/installation-registry-storage-non-production.adoc

```diff
@@ -8,7 +8,8 @@
 = Configuring storage for the image registry in non-production clusters
 
 You must configure storage for the image registry Operator. For non-production
-clusters, you can set the image registry to an empty directory.
+clusters, you can set the image registry to an empty directory. If you do so,
+then you cannot store images.
```

Comment:
If you do so, all images will be lost after the registry gets restarted.

Reply:
Thank you @dmage! I've updated the text.